src/functions/functionslog.ts

```diff
@@ -0,0 +1,51 @@
+import { logger } from "../logger";
+import { LogEntry } from "../gcp/cloudlogging";
+
+/**
+ * The correct API filter to use when GCFv2 is enabled and/or we want specific function logs
+ * @param v2Enabled check if the user has the preview v2 enabled
+ * @param functionList list of functions seperated by comma
+ * @returns the correct filter for use when calling the list api
+ */
+export function getApiFilter(v2Enabled: boolean, functionList?: string) {
```

Comment:
nit: my personal preference would be for the API signature to exist as if there was no preview flag and the code itself to work around the preview (e.g. so `getApiFilter(functionList?: boolean)` which then does the right thing depending on `previews.functionsv2`. That helps keep concerns stay as locally isolated as possible.

Reply:
Makes sense, fixed in the latest commit